lib/account.php

```diff
@@ -139,7 +139,7 @@ public function getImapConnection() {
 				'hostspec' => $host,
 				'port' => $port,
 				'secure' => $ssl_mode,
-				'timeout' => 20,
+				'timeout' => $this->config->getSystemValue('app.mail.imap.timeout', 20),
```

Comment:
it might make sense to cast to int here, like `'timeout' => (int) $this->config->getSystemValue('app.mail.imap.timeout', 20)`